src/go/pt-galera-log-explainer/types/timeline.go

```diff
@@ -139,7 +139,7 @@ func getlasttime(l LocalTimeline) time.Time {
 func CutTimelineAt(t LocalTimeline, at time.Time) LocalTimeline {
 	var i int
 	for i = 0; i < len(t); i++ {
-		if t[i].Date.Time.After(at) {
+		if t[i].Date != nil && t[i].Date.Time.After(at) {
```

Comment:
Are you sure that this is correct behavior. Cut timeline once we met nil value?

Reply:
This is the correct behavior: it's searching the first element which happened just after the date in parameter
It's just that few logs don't have a "date", so it's nil. It's only a silly mistake I'm fixing,` .Date` is a` *types.Date`

(they don't have a date because galera is making multiline logs, and multiline regexes are not worth it)